plenum/client/client.py

```diff
@@ -836,6 +853,23 @@ def verifyMerkleProof(*replies: Tuple[Reply]) -> bool:
                                                sha256_root_hash=rootHash))
         return True
 
+    def registerObserver(self, observer: Callable, name=None):
```

Comment:
Indy-node package uses this code extensively.

Reply:
Ok, but since we have these calls here it would be nice to have unit test here in plenum as well (not so critical, as this is a client code, which will be deprecated soon)

Reply:
I have added tests for this